README.md

```diff
@@ -17,11 +17,11 @@ We will be building an API for the purpose of accessing application data program
 Create `.env.test`:
 
 ```
-PGDATABASE=nc_games_test
+PGDATABASE=<test_database_name>
 ```
 
 Create `.env.development`:
```

Comment:
Id maybe just specify in the root of the project, but apart from that, all good.